src/components/pageTemplate/index.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+import AppBar from '@material-ui/core/AppBar';
+
+import Sidebar from '../sidebar';
+import Header from '../header';
+import useStyles from '../stylesPageTemplate';
+
+const PageTemplate = ({title, children}) => {
+  const pageTemplate = useStyles();
+  const [openSidebar, setOpenSidebar] = React.useState(false);
+
+  const handleSidebarOpen = () => {
+    setOpenSidebar(true);
+  };
+  const handleSidebarClose = () => {
+    setOpenSidebar(false);
+  };
+
+  return (
+      <div className={pageTemplate.root}>
+        <Header
+          openSidebar={openSidebar}
+          handleSidebarOpen={handleSidebarOpen}
```

Comment:
onSidebarOpen={handleSidebarOpen} and the same way in all project

Reply:
fixed